modules/chempy/cpv.py

```diff
@@ -23,41 +23,43 @@
 
 import math
 import random
-import copy
 
 RSMALL4 = 0.0001
 
 #------------------------------------------------------------------------------
-def get_null():
+def get_null() -> list[float]:
     return [0.0,0.0,0.0]
 
 #------------------------------------------------------------------------------
-def get_identity():
+def get_identity() -> list[list[float]]:
     return [[1.0,0.0,0.0],[0.0,1.0,0.0],[0.0,0.0,1.0]]
 
 #------------------------------------------------------------------------------
-def distance_sq(v1, v2):
+def distance_sq(v1: list[float], v2: list[float]) -> float:
```

Comment:
For arguments, I would use [Sequence](https://docs.python.org/3/library/collections.abc.html#collections.abc.Sequence) instead of `list`.